### ISSUE TITLE
Cannot install

### ISSUE THREAD

**Issue by narFnarF** [Dan-Q/twee2#16]
I'm new to Ruby so maybe this is a dumb question...

I installed Ruby and can run `ruby -v` without problem: `ruby 2.3.1p112 (2016-04-26 revision 54768) [i386-mingw32]`

Then, when I run `gem install twee2`, i get the following:

```
ERROR:  Could not find a valid gem 'twee2' (>= 0), here is why:
          Unable to download data from https://rubygems.org/ - SSL_connect returned=1 errno=0 state=
SSLv3 read server certificate B: certificate verify failed (https://api.rubygems.org/specs.4.8.gz)
```

I'm on Windows 7.

Any help?

**Comment by Dan-Q** [comment]
Looks like your copy of gem can't connect to https://rubygems.org/ because of an SSL error: most-likely, your computer does not trust the SSL certificate presented by rubygems.org. This isn't a Twee2 error, it's a RubyGems error.

A lazy workaround would be to allow gem to connect to rubygems.org without using SSL, e.g. by running:

    gem sources -a http://rubygems.org

(note the lack of "https" in my URL, above)

However, I wouldn't recommend making a habit of downloading and running Ruby gems over unencrypted connections. Instead, you should try to work out why you're getting problems verifying certificates.

[This comment](https://github.com/rubygems/rubygems/issues/1745#issuecomment-253739579) by @chris-morgan suggests the following fix:

> Copying https://raw.githubusercontent.com/rubygems/rubygems/master/lib/rubygems/ssl_certs/index.rubygems.org/GlobalSignRootCA.pem into C:\Ruby23-x64\lib\ruby\2.3.0\rubygems\ssl_certs solved the issue for me.

Note that the directory _you_ need to copy that file into may be different because you're using a different version of Ruby, but it should be similar.

I rarely use Ruby on Windows, but when I have, I've never experienced this issue. I've always installed Ruby for Windows, on the rare occasions I've used it, by using RubyInstaller (availale at https://rubyinstaller.org/) - I wonder if that might work for you, too? Just a thought.

In summary, three possible solutions are:

1. Download Ruby gems without encryption. (Eww.)
2. Fix your certificates. (See [this comment](https://github.com/rubygems/rubygems/issues/1745#issuecomment-253739579) for help.)
3. Install Ruby on Windows using RubyInstaller. (Maybe.)

**Comment by narFnarF** [comment]
Thanks for your help! Copying the .pem file that you suggested fixed the issue and I can now run twee2!
```
>twee2 version
   Your version: 0.5.0
 Latest version: 0.5.0
```

It's pretty weird though because I was actually using the latest RubyInstaller for windows, but it seems they are still on an old version and the certificate fix is in newer ruby versions.

Anyway, thanks for the help! So happy to be able to use twee again! :D
